intelmq/bots/parsers/cznic/parser_proki.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import json
+
+from intelmq.lib import utils
+from intelmq.lib.bot import ParserBot
+
+
+class CZNICProkiParserBot(ParserBot):
+
+    recover_line = ParserBot.recover_line_json
+
+    def parse(self, report):
+        raw_report = utils.base64_decode(report.get("raw"))
+        events = json.loads(raw_report).get("data")
```

Comment:
This can be problem when importing data from dump. If message could not be parsed, IntelMQ saves data to dump file, but in that file, message is stored as:

```json
[{"malware.name": "XXX", ...}]
```

So when recovering, this fails, because there is no `data` field.

```
2020-08-20 17:25:07,140 - proki-parser - ERROR - Bot has found a problem.
Traceback (most recent call last):
  File "/opt/dev_intelmq/intelmq/lib/bot.py", line 269, in start
    self.process()
  File "/opt/dev_intelmq/intelmq/lib/bot.py", line 968, in process
    for line in self.parse(report):
  File "/opt/dev_intelmq/intelmq/bots/parsers/cznic/parser_proki.py", line 14, in parse
    events = json.loads(raw_report).get("data")
AttributeError: 'list' object has no attribute 'get'
```